lldb/source/Core/Mangled.cpp

```diff
@@ -318,6 +322,26 @@ ConstString Mangled::GetDemangledName(// BEGIN SWIFT
       case eManglingSchemeD:
         demangled_name = GetDLangDemangledStr(mangled_name);
         break;
+#ifdef LLDB_ENABLE_SWIFT
+      case eManglingSchemeSwift: {
+        Log *log = GetLog(LLDBLog::Demangle);
+        LLDB_LOGF(log, "demangle swift: %s", mangled_name);
+        std::string demangled(SwiftLanguageRuntime::DemangleSymbolAsString(
+            mangled_name, SwiftLanguageRuntime::eTypeName, sc));
+        // Don't cache the demangled name the function isn't available yet.
+        if (!sc || !sc->function)
```

Comment:
- It doesn't look like you're using `sc` anywhere else in this block? 
- If you were to keep this, maybe move it to after the logging?